pkg/cmd/cli/cli.go

```diff
@@ -62,36 +62,40 @@ func NewCommandCLI(name, fullName string) *cobra.Command {
 	in := os.Stdin
 	out := os.Stdout
 
+	// OpenShift-specific commands
+	cmds.AddCommand(cmd.NewCmdBuildLogs(fullName, f, out))
+	cmds.AddCommand(cmd.NewCmdCancelBuild(fullName, f, out))
+	cmds.AddCommand(cmd.NewCmdConfig(fullName, "config"))
+	cmds.AddCommand(cmd.NewCmdDeploy(fullName, f, out))
+	cmds.AddCommand(cmd.NewCmdEdit(fullName, f, out))
+	cmds.AddCommand(cmd.NewCmdEnv(fullName, f, os.Stdin, out))
 	cmds.AddCommand(cmd.NewCmdLogin(fullName, f, in, out))
 	cmds.AddCommand(cmd.NewCmdLogout("logout", fullName+" logout", fullName+" login", f, in, out))
+	cmds.AddCommand(cmd.NewCmdNewApplication(fullName, f, out))
```

Comment:
No :)  See "rhc help" for what you're going to get :)

----- Original Message -----
> >  	cmds.AddCommand(cmd.NewCmdLogin(fullName, f, in, out))
> >  	cmds.AddCommand(cmd.NewCmdLogout("logout", fullName+" logout", fullName+"
> >  	login", f, in, out))
> > +	cmds.AddCommand(cmd.NewCmdNewApplication(fullName, f, out))
> 
> can we sort within group boundaries?  please? :)
> 
> 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1944/files#r30530947